docs/src/guides/spelling.md

```diff
@@ -0,0 +1,42 @@
+---
+icon: material/spellcheck
+---
+
+# Spell Checking
+
+## Introduction
+
+Spell checking is integrated into the [`check`](https://input-output-hk.github.io/catalyst-ci/onboarding/#pipeline) pipeline.
+This is to ensure the overall health of the project.
+
+This utilizes [`cspell`](cspell.org) under the hood for checking code and other text documents.
+It can be used to check for misspelled words, identify potential errors, and suggest corrections.
+
+## Configuration
+
+Each repo will need a [`cspell.json`](http://cspell.org/configuration/) file in the root of the repo.
+This file configures `cspell`.
+The file provided in the `Catalyst-CI` repo should be taken as a starting point
+for new projects.
+
+## Project Words
```

Comment:
I would also add a note that it is possible to extend this list of words for each specific file, which we are broadly using.
```
cspell: words <words>
```

Examples :
- https://github.com/input-output-hk/catalyst-ci/blob/c2f7d03c6c7c7ed5227e3398efab4b1d6c7d1508/earthly/docs/Earthfile#L3
- https://github.com/input-output-hk/catalyst-ci/blob/c2f7d03c6c7c7ed5227e3398efab4b1d6c7d1508/docs/src/guides/languages/postgresql.md#L12